### PR TITLE
Fixes #31999 - Debian remote strings for releases, components, et al

### PR DESCRIPTION
**Pull request by jjeffers**
These are separated by whitespace not commas.

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html [hunk 1]
```diff
@@ -122,7 +122,7 @@ <h4 translate> Sync Settings </h4>
                ng-model="repository.deb_releases"
                type="text"/>
         <p class="help-block" translate>
-          Comma separated list of releases (suite or codename) to sync from. Default: stable
+          Whitespace separated list of releases (suite or codename) to sync from. Default: stable
```
Comment (ianballou) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html:125]: While we're editing these lines, let's add the dash "Whitespace-separated".

**Review by ianballou — COMMENTED** [review]
Just one comment, looks good otherwise.

**Review by ianballou — APPROVED** [review]
Looks good to me!

**Comment by ianballou** [comment]
I tested the whitespace in all 3 fields and synced. Worked fine.

Fields:
```
Upstream URL
    http://ftp.debian.org/debian
Releases
    stable-updates jessie-updates
Components
    contrib main
Architectures
    amd64 arm64 
```

**Review by chris1984 — APPROVED** [review]

**Pull request merged**